config/stack_noaa.yaml

```diff
@@ -53,25 +53,25 @@ netcdf:
   version_cxx: 4.3.1
 
 nccmp:
-  build: YES
+  build: NO
```

Comment:
please revert to develop

Reply:
I made this change, however I did not test with this value set, so I will need to retest once all of these changes are in place.